projects/arrow/project.yaml

```diff
@@ -14,4 +14,5 @@ auto_ccs:
   - "wesmckinn@gmail.com"
   - "xhochy@gmail.com"
   - "will.jones127@gmail.com"
+  - "maplewish117@gmail.com"
```

Comment:
Can you keep this list in alphabetical order?

Reply:
done